app/interactions/pulitzer/content_elements_controller/update.rb

```diff
@@ -0,0 +1,21 @@
+class Pulitzer::ContentElementsController::Update
+
+  def initialize(content_element, params)
+    @content_element, @params = content_element, params.to_h.to_hash.symbolize_keys
```

Comment:
Because of the delete below, I *think* we could be removing from the original params hash. If that's not intentional, we should probably `.dup` it before messing with it.